tests/platform_tests/broadcom/files/ser_injector.py

```diff
@@ -94,6 +247,31 @@ def run_cmd(cmd):
     stdout, stderr = out.communicate()
     return stdout, stderr
 
+def get_asic_type():
```

Comment:
can you move this to a helper? might be useful for other tests as well.

Reply:
I can add one with another PR. but this is in the file executed on the DUT. It is standalone script.

Reply:
ok, scratch the another PR part. I added the helper function, also noticed that asic_type was used already. So I name the function get_asic_name() instead.